index.js

```diff
@@ -1 +1,37 @@
-console.log("Hello World!");
+const distanceFunctions = require('./distance_calculation.js');
+const usersFunctions = require('./users_operation.js');
+const officeCordsLat = 42.6665921;
+const officeCordsLon = 23.351723;
+const distanceinKmRequiredForInvitation = 100.0;
+
+const fs = require("fs");
+const readline = require("readline");
+
+let usersAndTheirDistanceFromOffice = [];
+let usersFinalListForInvitation = [];
```

Comment:
Since these are arrays, you can declare them with const instead of let. https://www.tutorjoes.in/JS_tutorial/use_of_const_for_creating_arrays_in_javascript